docker-compose.yml

```diff
@@ -1,5 +1,17 @@
 version: '3'
 services:
+services:
+  nginx:
+    image: nginx:latest
+    restart: always
+    ports:
+      - 80:80
+    volumes:
+      - ./exp/psiturk-example:/var/www/psiturk-example:ro
```

Comment:
can you map `./exp` to `/var/www` instead (don't know if it will work)? I often have a bunch of folders inside `./exp` for the different experiments, so I wonder if this will make it work.

Reply:
@mvdoc Are you often *serving* multiple experiments at the same time, or do you usually just have multiple experiments in the same folder?

You certainly can map `./exp` to `/var/www`, but then you'll need to change the `root` specification in `default.conf` to point to the experiment that you want to serve, at any given time.

I suppose in this particular case, you wouldn't need to change anything in `default.conf`.

If, however, you actually want to serve multiple experiments at the same time (which I'm guessing you're not doing?), you'll need more than one `server` block in the `default.conf` file, and you'll need some way of distinguishing them, either via the port or via the `server_name` (but then you'd probably need to use, *e.g.*, subdomains of a domain you own). (In this case, I suppose you'd also need to be changing the port number for the different psiturk instances, which is why I assume you're probably not doing this.)

Reply:
No, you're definitely right that I'm not serving multiple experiments at the same time. However, it seems to me that now we need to modify both the `default.conf` and the `docker-compose.yml` every time we want to serve a different experiment (i.e., a different folder within `./exp`), or am I missing something?

Reply:
Yes, that's right. So yes, your suggestion of mapping `./exp` to `/var/www` is probably better if you have multiple experiments. That's not a situation I was thinking about. If you did that, then all you would have to do is modify `default.conf` when you want to serve a different experiment. Shall I change it in the PR?

Reply:
Yes please. In that way users need to modify only one file (`default.conf`) instead of two :)